src/main/scala/com/mwz/sonar/scala/util/util.scala

```diff
@@ -19,56 +19,38 @@
 package com.mwz.sonar.scala
 package util
 
-import java.io.File
 import java.nio.file.{Path, Paths}
 import java.util.Optional
 
-import org.sonar.api.batch.fs.{FileSystem, InputFile}
-import org.sonar.api.batch.measure.Metric
-import org.sonar.api.batch.sensor.SensorContext
-import org.sonar.api.config.Configuration
+import org.sonar.api.batch.fs.FileSystem
 
-import scala.language.{higherKinds, implicitConversions}
-import scala.util.Try
+import scala.language.implicitConversions
 
 /**
  *  Scala.Option <-> Java.Optional conversions.
  *  @note Taken from https://gist.github.com/julienroubieu/fbb7e1467ab44203a09f.
  */
 object JavaOptionals {
-  implicit def toRichOption[T >: Null](opt: Option[T]): RichOption[T] = new RichOption[T](opt)
-  implicit def toRichOptional[T](optional: Optional[T]): RichOptional[T] = new RichOptional[T](optional)
+  implicit final def toRichOption[T >: Null](opt: Option[T]): RichOption[T] =
```

Comment:
Since the method is part of an `object` and an `object` is already _final_...
marking a `def` _final_ is not redundant? - or does it help the JIT inlining or something like that?

Reply:
This should be a trait (as per the examples above), but I haven't finished refactoring everything yet 😄

Reply:
Oh that makes sense.